### PR TITLE
[CFU] Add tests for multiple choice summary

### PR DESCRIPTION
**Pull request by rshipp**
Followup to #51653: Add eyes and unit tests for new multiple-choice summary features.

Ran the eyes tests locally through sc-tunnel and accepted the baselines, so hopefully it'll pass dtt first-try.

## Links

- jira ticket: [TEACH-469](https://codedotorg.atlassian.net/browse/TEACH-469)

### CODE REVIEW

File: apps/test/unit/templates/levelSummary/MultiResponses.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    expect(wrapper.find('Chart').prop('data')).to.eql([
+      ['Answer', 'Count', {role: 'annotation'}, {role: 'style'}],
+      ['A', 0, '0', null],
+      ['B', 1, '1', null],
+    ]);
+  });
+});
+
+describe('multiAnswerCounts', () => {
+  it('defaults to zero with 4 answers', () => {
```
Comment (hannahbergam) [inline apps/test/unit/templates/levelSummary/MultiResponses.jsx:33]: Curious about the difference between 4 and 6!

File: apps/test/unit/templates/levelSummary/SummaryResponses.jsx [hunk 2]
```diff
@@ -108,4 +108,57 @@ describe('SummaryResponses', () => {
...
+          },
+        },
+        answer_is_visible: true,
+      }
+    );
+
+    expect(wrapper.find('ToggleSwitch').length).to.eq(0);
+  });
+
+  it('does not render toggle without policy permission', () => {
```
Comment (hannahbergam) [inline apps/test/unit/templates/levelSummary/SummaryResponses.jsx:148]: I'm curious about how we can tell if it should render here or not!

File: dashboard/test/ui/features/teacher_tools/level_summary.feature [hunk 3]
```diff
@@ -33,3 +34,31 @@ Scenario: Free Response level 3
...
+  When I open my eyes to test "multi summary 1"
+  Given I create a teacher named "Teacher_1"
+  And I give user "Teacher_1" authorized teacher permission
+  And I create a new student section
+  And I am on "http://studio.code.org/s/allthethings/lessons/9/levels/1/summary"
+  And I wait until element "#summary-container" is visible
+  And I wait to see ".uitest-sectionselect"
+  Then I see no difference for "multi level summary 1"
+  And I click selector "button.toggle-input"
+  And I see no difference for "multi level summary 1 show answer"
```
Comment (hannahbergam) [inline dashboard/test/ui/features/teacher_tools/level_summary.feature:49]: Yessss I love a good eyes test

**Review by hannahbergam — COMMENTED** [review]
Just a couple questions! Additionally- do we want to have a quick eyes test for how it looks when there are no responses?

File: apps/test/unit/templates/levelSummary/MultiResponses.jsx (hunk 1 above)
Comment (rshipp) [reply apps/test/unit/templates/levelSummary/MultiResponses.jsx:33]: when I started I assumed there would always be 4, but then I found out there can be any number of answers. so i just wanted to have a test for the most common case of 4 answers, and one that had more than 4 answers. the 6 is arbitrary.

File: apps/test/unit/templates/levelSummary/SummaryResponses.jsx (hunk 2 above)
Comment (rshipp) [reply apps/test/unit/templates/levelSummary/SummaryResponses.jsx:148]: without `answer_is_visible: true,` from the policy, it doesn't render. this was a bugfix introduced in #51792.

**Review by hannahbergam — APPROVED** [review]
LGTM! Thanks for following up

**Comment by rshipp** [comment]
> do we want to have a quick eyes test for how it looks when there are no responses?

whoops i missed this question. all the eyes tests are with no responses, unless I'm misunderstanding the question. i didn't think it was worth it to try and populate student answers from the eyes tests.

**Pull request merged**